source/h2_stream.c

```diff
@@ -706,7 +713,11 @@ int aws_h2_stream_on_activated(struct aws_h2_stream *stream, enum aws_h2_stream_
         AWS_H2_STREAM_LOGF(ERROR, stream, "Failed to create HEADERS frame: %s", aws_error_name(aws_last_error()));
         goto error;
     }
-
+    if (stream->base.on_metrics) {
+        stream->base.metrics.stream_id = stream->base.id;
+        AWS_ASSERT(stream->base.metrics.send_start_timestamp_ns == 0);
+        aws_high_res_clock_get_ticks(&stream->base.metrics.send_start_timestamp_ns);
+    }
```

Comment:
Similar to h1, this seems misleading. We haven’t actually started sending the data, only scheduled it.

Reply:
I updated the documentation, https://github.com/awslabs/aws-c-http/pull/433/files#r1163348036